packages/fast-components-react-base/src/select/select.tsx

```diff
@@ -464,20 +464,20 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, Select
         if (props.multiselectable) {
             return null;
         }
+
         const isItemSelected: boolean = state.selectedItemIndex !== 0;
         return (
             <button
-                disabled={props.disabled}
                 id={triggerId}
-                role="option"
-                aria-atomic={true}
-                aria-label={state.displayString}
-                aria-expanded={state.isMenuOpen}
-                aria-selected={isItemSelected}
-                aria-posinset={isItemSelected ? state.selectedItemIndex : null}
-                aria-setsize={isItemSelected ? state.selectableItemCount : null}
+                disabled={props.disabled}
+                aria-disabled={this.props.disabled}
+                aria-expanded={this.state.isMenuOpen}
+                aria-labelledby={this.props.labelledBy || null}
+                aria-multiline={false}
+                value={state.displayString}
+                role="textbox"
```

Comment:
Why is this a button element if it's a textbox? Could it be an input?

Reply:
Basically Aria does not have good way to describe this control (silly but true).  We wanted the act of focusing on a collapsed select component to cause screen readers to enumerate "control label", "current value", "control type", "iscollapsed".  The only component that does this in the widget list (https://www.w3.org/TR/wai-aria-1.1/#widget_roles) turned out to be combobox.  We tried menu, but that doesn't enumerate value.  Aria combo boxes expect to own an element with role of "textbox" and need that in order to have something to extract the current value from.  So no, we should not make it actually be an input as we don't actually want that functionality, but we should have our accessibility folks raise this with the spec making folks at W3.

Reply:
Without context in the code there's no way for anyone to know that, so could you drop a comment in here explaining why we are using a combobox when this is not actually one.

Reply:
Added some comments.